server/src/main/scala/io/delta/common/CloudFileSigner.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.delta.sharing.server
+package io.delta.common
```

Comment:
should it be io.delta.sharing.server.common?

Reply:
fixed